doc/user/content/sql/create-source/postgres.md

```diff
@@ -431,6 +433,8 @@ CREATE SOURCE mz_source
 
 ### Handling errors and schema changes
 
+{{< include-md file="shared-content/schema-changes-in-progress.md" >}}
```

Comment:
https://preview.materialize.com/materialize/32436/sql/create-source/postgres/#handling-errors-and-schema-changes